tests/functional/disaster-recovery/regional-dr/test_failover_and_relocate_discovered_apps.py

```
@@ -31,353 +32,285 @@ class TestFailoverAndRelocateWithDiscoveredApps:
-                            namespace=rdr_workload.workload_namespace,
-                            should_exist=True,
-                        )
-                        dr_helpers.wait_for_resource_count(
-                            kind=constants.VOLUMESNAPSHOT,
-                            namespace=rdr_workload.workload_namespace,
-                            expected_count=rdr_workload.workload_pvc_count,
-                        )
-                    wait_for_replication_destinations_creation(
-                        rdr_workload.workload_pvc_count, rdr_workload.workload_namespace
-                    )
+        for drpc_obj, rdr_workload in zip(drpc_objs, rdr_workloads):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)